### PR TITLE
Update for Sass Linter that doesn't require Ruby

### PR DESCRIPTION
**Pull request by phuson**
scss-lint (ruby dependency) ➡️  sass-lint (node only)

Please note that the conversion from `.scss-lint.yml` to `.sass-lint.yml` is not entirely 1 to 1. Here's the converter that I used: http://sasstools.github.io/make-sass-lint-config/

Reference Issue: https://github.com/grommet/grommet/issues/373

### CODE REVIEW

File: src/gulp-tasks-linters.js [hunk 1]
```diff
@@ -3,18 +3,19 @@ import fs from 'fs';
 import eslint from 'gulp-eslint';
 import shelljs from 'shelljs';
```
Comment (alansouzati) [inline src/gulp-tasks-linters.js]: I believe we can remove the shelljs dependency.

**Comment by alansouzati** [comment]
Thanks for this change. very important one as now we will be a node-only project.

I've sent a minor suggestion where I believe shelljs is not needed anymore and can also be removed as a dependency from grommet-toolbox.

**Comment by phuson** [comment]
Good catch. Removed `shelljs` dependency. Thanks!

**Comment by alansouzati** [comment]
Thanks for your contribution!

**Pull request merged**